src/main/java/com/spotify/docker/BuildMojo.java

```diff
@@ -409,11 +429,22 @@ private void loadProfile() throws MojoExecutionException {
       exposesSet.add(expand(raw));
     }
 
+    List<String> runListTmp = emptyList();
+    try {
+      runListTmp = profileConfig.getStringList("runs");
```

Comment:
Would it be possible to just do `runList.addAll()` in here and get rid of the `runListTmp` entirely?